lib/Sema/CSApply.cpp

```diff
@@ -1287,12 +1287,14 @@ namespace {
                          ArrayRef<Identifier> argLabels,
                          bool hasTrailingClosure,
                          ConstraintLocatorBuilder locator,
-                         bool isImplicit, AccessSemantics semantics) {
+                         bool isImplicit, AccessSemantics semantics,
+                         Optional<SelectedOverload> selected = None) {
```

Comment:
is this ever going to be None?

Reply:
Two of the three clients pass none here, so yep.